awx/main/models/ha.py

```diff
@@ -92,6 +92,10 @@ class Instance(HasPolicyEditsMixin, BaseModel):
         default=0,
         editable=False,
     )
+    is_containerized = models.BooleanField(
```

Comment:
Do we need an actual field for this now, since under the current model, it's *always* `False`?

Seems like we could just lose this field and know that `Instance` objects _always_ represent an actual VM/machine (not a pod).

Reply:
Good catch, I added this and never went anywhere with it.